lib/ui/painting/canvas.cc

```diff
@@ -198,7 +198,6 @@ void Canvas::clipPath(const CanvasPath* path, bool doAntiAlias) {
         ToDart("Canvas.clipPath called with non-genuine Path."));
     return;
   }
-  external_allocation_size_ += path->path().approximateBytesUsed();
```

Comment:
Will the engine no longer be reporting these sizes as the external allocation size for Dart objects with native fields? Or does this only affect HintFreed? (Same question here and below.)

Reply:
Correct. I can split this into another patch, but basically this seems like it was a mistake. We've already moved the reporting of images for this.

The basic problem is that from scene to scene, we might be reusing these things, and end up reporting vastly inflated numbers to the VM, and then that creates lots of wasteful garbage collections.